action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -481,6 +485,22 @@ func (stateDB *StateDBAdapter) HasSelfDestructed(evmAddr common.Address) bool {
 	return ok
 }
 
+// Selfdestruct6780 implements EIP-6780
+func (stateDB *StateDBAdapter) Selfdestruct6780(evmAddr common.Address) {
```

Comment:
it's differ from implementation in geth, pls double-check:
https://github.com/ethereum/go-ethereum/blob/v1.13.14/core/state/statedb.go#L459

Reply:
discussed offline